apps/web/src/pages/templates/editor/TemplateEditorPage.tsx

```diff
@@ -21,8 +22,8 @@ export default function TemplateEditorPage() {
   const handleAddChannel = (tabKey) => {
     const foundChannel = channelButtons.find((item) => item === tabKey);
     if (!foundChannel) {
-      toggleChannel(ChannelTypeEnum[tabKey], true);
-
+      // toggleChannel(tabKey, true);
```

Comment:
```suggestion
```
Is this not needed anymore? If not maybe better to remove it?

Reply:
yep, missed it